### ISSUE TITLE
-Pw server switch fails on ban vote 

### ISSUE THREAD

**Issue by ghost** [assaultcube/AC#9]
In my servercmdline.txt I have -PfKbMAsRCDEw

the 'w' part does this:

[quote]w vote to kick/ban for "weak" reasons, i.e, the other player is not obviously lagging, nor teamkilling, nor spamming, nor cheating (as far as the server can detect it)[/quote]

As it is set to lowercase, I should be able to kick/ban as a normal user OR admin for any reason. I can kick as a normal user or admin, but I can only ban as an admin. When I try to ban as a normal user, I get "could not vote: server denied your call" UNLESS the person I am banning has done something the server considers bannable, such as spamming, TKing, too high score ratio, etc.

In other words, it is acting like I used -PW instead of -Pw when I try to do a ban vote.
